lib/lambda.d/tunasync-handler/index.py

```diff
@@ -0,0 +1,37 @@
+import json
+import urllib3
+import os
+import logging
+
+logger = logging.getLogger()
+logger.setLevel(logging.INFO)
+http = urllib3.PoolManager()
+tunasync_manager_url = os.environ['TUNASYNC_MANAGER_URL']
+
+def handler(event, context):
+    logger.info(event)
+    requestUrl = tunasync_manager_url + '/cmd'
+
+    requestBody = {}
+    requestBody['cmd'] = 0 # The first command is CmdStart, see CmdStart in tunasync/internal/msg.go
```

Comment:
is possible using cmd name `restart` in case the order is changed in upstream `tunasync`?

Reply:
I will submit a pr for this change.

Reply:
let’s track the possible improvement in issue #112